src/app/modules/group/components/group-links/group-links.component.html

```diff
@@ -13,6 +14,6 @@
     >, and</span>
   </span>
   <span class="more" *ngIf="(items || []).length > maxItemsDisplay && isLast" i18n>
-    {{ (items || []).length - maxItemsDisplay }} more
+    {{ (items || []).length - ((items || []).length > maxItemsDisplay ? maxItemsDisplay - 1 : maxItemsDisplay) }} more
```

Comment:
`(items || []).length > maxItemsDisplay ? ... : ...`  aren't you in case where this is always true?

Reply:
Items (groups) can be 1, can be 6 and more, maxItemsDisplay equal 4. So I guess it shouldn't be always true. I have tested different cases

Reply:
line 16 is ` *ngIf="(items || []).length > maxItemsDisplay &&  ...` 🤨

Reply:
That condition does:

If groups count more than 4, then need to show "more" caption after last group, but if there are less or equal 4 groups, then no need to show "more" caption.

Reply:
If line 16 is that L17 is displayed if "`(items || []).length > maxItemsDisplay` and something... we are sure on line 17 that `(items || []).length > maxItemsDisplay` is true... so the ternary operation `(items || []).length > maxItemsDisplay ? ... : ...` is useless (it is always true).

I don't try to get the business logic, just programming logic.

Reply:
Got it, Fixed